rules.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
     "import/no-default-export": 2,
     "kodekonveyor/no-comment": 2,
-    "promise/prefer-await-to-then": "error"
+    "promise/prefer-await-to-then": "error",
+    "filenames/match-regex": [2, "(^[a-z]+[A-Z]+[a-z]+[.](?:tsx)$).*", true],
```

Comment:
I get this error when linting:
Definition for rule 'filenames/match-regex' was not found     filenames/match-regex

I did add "eslint-plugin-filenames": "^1.3.2", to devdependencies of the project.

Reply:
https://stackoverflow.com/questions/40834950/eslint-configuration-is-showing-a-definition-for-rule-filenames-match-regex-w